libs/community/langchain_community/chat_models/sparkllm.py

```diff
@@ -224,6 +224,11 @@ def validate_environment(cls, values: Dict) -> Dict:
             "IFLYTEK_SPARK_LLM_DOMAIN",
             SPARK_LLM_DOMAIN,
         )
+        return values
+
+    @root_validator(pre=False, skip_on_failure=True)
+    def post_init(cls, values: Dict) -> Dict:
+        """Post init validation for the class."""
         # put extra params into model_kwargs
         values["model_kwargs"]["temperature"] = values["temperature"] or cls.temperature
```

Comment:
```suggestion
        values["model_kwargs"]["temperature"] = values["temperature"]
```


`cls.temperature` doesn't exist I think

Reply:
you are right. but same the below, it should  move back the pre=True? it didn't have any checks.